test/bin/test-citgm.js

```diff
@@ -54,6 +54,30 @@ test('bin: omg-i-pass /w custom script /w tap to file /w junit to file /w append
   });
 });
 
+test('bin: omg-i-pass /w custom test script passing', function (t) {
+  t.plan(1);
+  var proc = spawn(citgmPath, ['omg-i-pass', './test/fixtures/example-test-script-passing.sh']);
```

Comment:
can you do this with `omg-i-fail` passing just so we explicitly know we won't get a false positive